### PR TITLE
Scikit-learn 1.2.2 docset

### PR DESCRIPTION
**Pull request by capac**
Updated scikit-learn docset to 1.2.2.

### CODE REVIEW

**Comment by Kapeli** [comment]
This pull request seems to also update statsmodels/statsmodels.tgz. Is this intentional?

File: docsets/Scikit/docset.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "name": "scikit-learn",
-    "version": "1.1.2",
+    "version": "1.2.2",
     "archive": "scikit-learn.tgz",
```
Comment (Kapeli) [inline docsets/Scikit/docset.json:4]: Please update the archive at this path. You added `scikit-learn-1.2.2.tgz` instead.

**Comment by capac** [comment]
I have updated the archive to Scikit-Learn 1.2.2. The PR also contains the statsmodels 0.14 package.

**Comment by capac** [comment]
I had left statsmodels.tgz in my local repository and it was push together with the Scikit-learn 1.2.2 docset. This is the same statsmodels package from a previous PR of mine. You may remove it. Sorry for the confusion.

**Comment by Kapeli** [comment]
Thank you!

**Pull request merged**